code/modules/mob/living/carbon/xenomorph/abilities/general_powers.dm

```diff
@@ -27,8 +27,16 @@
 		return
 
 	var/obj/effect/alien/weeds/node/N = locate() in T
-	if(N && N.weed_strength >= X.weed_level)
-		to_chat(X, SPAN_WARNING("There's a pod here already!"))
+	if(N)
+		if(N.weed_strength > X.weed_level)
+			to_chat(X, SPAN_WARNING("There's a pod here already!"))
+		else
+			to_chat(X, SPAN_WARNING("You start removing the resin node."))
+			if(!do_after(X, 10, INTERRUPT_ALL|BEHAVIOR_IMMOBILE, BUSY_ICON_BUILD))
```

Comment:
Use the time macros. E.g. `1 SECONDS` See https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STYLES.md#use-our-time-defines